src/patterns/organisms/split/split.hbs

```diff
@@ -0,0 +1,36 @@
+---
+name: Split
+order: 1
+description: |
+  Split a row into text and a single piece of media (image or video).
+notes: |
+  - Highly customizable positioning options
```

Comment:
We should list all the option classes here, but we can expand docs later. Not a blocker.